### PR TITLE
(TASK-65) DeveloperExceptionHandlerMiddleware throws an Exception

### PR DESCRIPTION
**Pull request by smrsgv**
Keep in mind you can change behavior (response) for production and development environments in launchSettings.json by setting `"ASPNETCORE_ENVIRONMENT": "Production"`

### CODE REVIEW

**Comment by abbcyhn** [comment]
@smrsgv PL name is not correct. Fix it.

File: WebAPI/Middlewares/DeveloperExceptionHandlerMiddleware.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public async Task InvokeAsync(HttpContext context, RequestDelegate next)
                     Status = StatusCodes.Status500InternalServerError,
                     Type = "https://datatracker.ietf.org/doc/html/rfc7231#section-6.6.1",
                     Title = e.Message,
-                    Detail = e.InnerException!.Message + Environment.NewLine + e.StackTrace,
+                    Detail = e.InnerException?.Message + Environment.NewLine + e?.StackTrace,
```
Comment (abbcyhn) [inline WebAPI/Middlewares/DeveloperExceptionHandlerMiddleware.cs:22]: It would be easy to fix if you tested your changes!

Comment (smrsgv) [reply WebAPI/Middlewares/DeveloperExceptionHandlerMiddleware.cs:22]: Definitely you are right Sir!

**Comment by smrsgv** [comment]
> @smrsgv PL name is not correct. Fix it.

Do you mean PR (pull request) name? Already fixed.

**Review by abbcyhn — APPROVED** [review]

**Pull request merged**